lib/ui/painting.dart

```diff
@@ -1741,6 +1741,16 @@ class ImageFilter extends NativeFieldWrapperClass2 {
     _initBlur(sigmaX, sigmaY);
   }
   void _initBlur(double sigmaX, double sigmaY) native 'ImageFilter_initBlur';
+
+  /// Creates an image filter that applies a matrix transformation.
```

Comment:
Can we add a little more colour here, e.g. give an example of a matrix and the effect it would have? I think that would be helpful for people subsequently looking at this, especially if we also add other kinds of matrix image filters (e.g. we don't want people confusing this with a kernel image filter).

Reply:
e.g. if a Matrix4.scale(2) matrix causes the background to be magnified by 2, we could say `For example, applying a positive scale matrix (see [new Matrix4.scale]) would magnify the background when a paint operation with this filter is drawn on the canvas.`

Reply:
done